fractal_tasks_core/tasks/illumination_correction.py

```diff
@@ -198,9 +194,7 @@ def illumination_correction(
     FOV_ROI_table = ad.read_zarr(f"{zarrurl_old}/tables/FOV_ROI_table")
 
     # Read pixel sizes from zattrs file
-    full_res_pxl_sizes_zyx = extract_zyx_pixel_sizes(
-        f"{zarrurl_old}/.zattrs", level=0
-    )
+    full_res_pxl_sizes_zyx = ngff_image_meta.get_pixel_sizes_zyx(level=0)
```

Comment:
Can we take this up to line 180 with the rest of the metadata extraction?

Reply:
Fixed with https://github.com/fractal-analytics-platform/fractal-tasks-core/pull/528/commits/0674165c32c4521682ad505cca7d9c2aba3726ad